shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -197,42 +231,20 @@ - (void)clipPath:(const SkPath&)path {
     }
     verb = iter.next(pts);
   }
-
-  CAShapeLayer* clip = [[[CAShapeLayer alloc] init] autorelease];
-  clip.path = pathRef;
-  self.layer.mask = clip;
+  paths_.push_back([self getTransformedPath:pathRef matrix:matrix]);
 }
 
-- (void)setClip:(flutter::MutatorType)type
-           rect:(const SkRect&)rect
-          rrect:(const SkRRect&)rrect
-           path:(const SkPath&)path {
-  FML_CHECK(type == flutter::clip_rect || type == flutter::clip_rrect ||
-            type == flutter::clip_path);
-  switch (type) {
-    case flutter::clip_rect:
-      [self clipRect:rect];
-      break;
-    case flutter::clip_rrect:
-      [self clipRRect:rrect];
-      break;
-    case flutter::clip_path:
-      [self clipPath:path];
-      break;
-    default:
-      break;
-  }
+- (fml::CFRef<CGPathRef>)getTransformedPath:(CGPathRef)path matrix:(CATransform3D)matrix {
+  CGAffineTransform affine =
+      CGAffineTransformMake(matrix.m11, matrix.m12, matrix.m21, matrix.m22, matrix.m41, matrix.m42);
+  CGPathRef transformedPath = CGPathCreateCopyByTransformingPath(path, &affine);
+  CGPathRelease(path);
+  return fml::CFRef<CGPathRef>(transformedPath);
 }
 
-// The ChildClippingView is as big as the FlutterView, we only want touches to be hit tested and
-// consumed by this view if they are inside the smaller child view.
-- (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
-  for (UIView* view in self.subviews) {
-    if ([view pointInside:[self convertPoint:point toView:view] withEvent:event]) {
-      return YES;
-    }
-  }
-  return NO;
+- (CGRect)getCGRectFromSkRect:(const SkRect&)clipSkRect {
+  return CGRectMake(clipSkRect.fLeft, clipSkRect.fTop, clipSkRect.fRight - clipSkRect.fLeft,
+                    clipSkRect.fBottom - clipSkRect.fTop);
 }
 
 @end
```

Comment:
gpus_ReturnNotPermittedKillClient crash on iOS [62403](https://github.com/flutter/flutter/issues/62403) Can you help us have a look，please？

Reply:
The issue doesn't seem to related to this PR?